tests/testthat/test-derive_param_clinbenefit.R

```diff
@@ -96,6 +96,7 @@ test_that("Clinical benefit rate parameter is derived correctly", {
     source_datasets = list(adrs = adrs),
     reference_date = TRTSDT,
     ref_start_window = 28,
+    clinben_vals = c("CR", "PR", "SD"),
```

Comment:
@mattmarino @bundfussr in my mind there's no benefit in adding this unless you also add some input test cases that actually are affected by this so that you can test the new functionality - would  you agree? in this case there are no `NON-CR/NON-PD` cases so this line makes zero impact and therefore we don't get the new functionality auto-tested in any way.

Reply:
@rossfarrugia , you are right, it does not make a difference. We could improve the testing and add more cases but I am not sure if we should do it now. Seems not essential to me.

Reply:
i think adding 1 extra test subject could solve this and shouldn't take too long? to save us coming back to later. i can try and add this now quickly before my morning full of meetings start...

Reply:
Thanks